src/kernels/errors/kernelErrorHandler.web.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { injectable } from 'inversify';
+import { Resource } from '../../platform/common/types';
+import { DataScienceErrorHandler } from './kernelErrorHandler';
+
+@injectable()
+export class DataScienceErrorHandlerWeb extends DataScienceErrorHandler {
+    protected override async addErrorMessageIfPythonArePossiblyOverridingPythonModules(
+        _messages: string[],
+        _resource: Resource
+    ) {
+        //
```

Comment:
I wonder if we need a remote check in the node version? Theoretically there could be a remote version of this too where we use the cwd of the kernel process to check for files (instead of reading from disk)

Reply:
> we need a remote check in the node version? Theoretically there could be a remote version o

**Agreed, however** 
* I'm assuming things are already working on the remote because they already have jupyter server running
* reading the files via a kernel would still require the kernel to start (but this code is used to detect what could cause the kernel to fail)


I think this will be necessary when users have access to some kind of a file system where they can create files, and at that point I think we can try to look into this. Else I feel we can leave this for now (at least for remote).

Reply:
Okay maybe not the cwd of the kernel process, but rather the file system api from the Jupyter server.